src/Initializer/InitProcedure/InitMesh.cpp

```diff
@@ -58,24 +59,26 @@ static void readMeshPUML(const seissol::initializer::parameters::SeisSolParamete
   const int rank = seissol::MPI::mpi.rank();
   double nodeWeight = 1.0;
 
-#ifdef USE_MINI_SEISSOL
-  if (seissol::MPI::mpi.size() > 1) {
-    logInfo(rank) << "Running mini SeisSol to determine node weight";
-    auto elapsedTime = seissol::miniSeisSol(seissol::SeisSol::main.getMemoryManager(),
-                                            seissolParams.model.plasticity);
-    nodeWeight = 1.0 / elapsedTime;
-
-    const auto summary = seissol::statistics::parallelSummary(nodeWeight);
-    logInfo(rank) << "Node weights: mean =" << summary.mean << " std =" << summary.std
-                  << " min =" << summary.min << " median =" << summary.median
-                  << " max =" << summary.max;
-
-    writer::MiniSeisSolWriter writer(seissolParams.output.prefix.c_str());
-    writer.write(elapsedTime, nodeWeight);
+  if (utils::Env::get<bool>("SEISSOL_MINISEISSOL", true) && seissolParams.mesh.miniseissol) {
```

Comment:
why to read the environment variable, if miniseissol is defined in the parameter file?
I would only keep the parameter file.
(by the way, commthread could also be defined in the parameter file).

Reply:
Personally, I'd tend to keep them an environment variable at the moment, because they don't influence the simulation per se.
The mini SeisSol got into the parameter file as well, because it seemed to be needed in #427 IMO.

Reply:
I don't understand. I would only keep the environement variable, and remove seissolParams.mesh.miniseissol.
#427 can be fixed by using the environement variable, right?

Reply:
Yup, it can. Ok, I'll remove the parameter from the parameter file.